src/options.js

```diff
@@ -59,6 +59,12 @@ function normalize(options) {
     normalized.trailingComma = "none";
   }
 
+  // TODO: check if this changes users' options
+
+  if (normalized.parser === "python") {
+    normalized.tabWidth = 4;
```

Comment:
There might be a case to _only_ support tabWidth of 4 in Python. (e.g. say that this setting does not apply to Python)

Reply:
Google’s Python style guide says 2 spaces.

Reply:
Oh wow. Google doesn't follow PEP-8?

Reply:
The internal Google Python style guide mandates 2 spaces. You can see it in a number of their open-source projects: https://github.com/tensorflow/tensorflow/blob/v1.4.0/tensorflow/python/ops/nn_ops.py. It's real fun when you're backporting stuff into your own code base that is otherwise getting linted to ensure 4 spaces.

And of course if you've done anything with numpy, it's impossible to follow PEP 8 there, since implicit boolean casts for numpy arrays just raise exceptions.

And, like, `logging` doesn't even follow the camel_case thing. So, whatever... lol PEP 8? I dunno.

Reply:
Please don't force tabWidth 4 for Python.. 2 is cool as well :)

Reply:
totally, when I did that the idea was to "force" 4 as the default, but then you could change it, I'll make sure that this will be the case :)

Reply:
Maybe something like this then:

```js
    if (normalized.parser === "python" && !normalized.tabWidth) {
      normalized.tabWidth = 4;
    }
```